core/common/src/ChangesetProps.ts

```diff
@@ -100,3 +102,31 @@ export interface ChangesetRange {
   /** index of last changeset. If undefined, all changesets after first are returned. */
   end?: ChangesetIndex;
 }
+
+/**
+ * Statistics for a single SQL statement executed during changeset application.
+ * @public
```

Comment:
do you really want these public right as we add them?

theres `@public @preview` and `@beta`